Bookish.DataAccess/BookishService.cs

```diff
@@ -87,5 +90,39 @@ ORDER BY
 
             return dbConnection.Query<BookLoan>(query, new { UserID = userId });
         }
+
+        public void AddBook(string title, string author, string isbn, int totalCopies)
+        {
+            var lastId = LastCopiesId();
```

Comment:
Why do you need this? You don't need to insert the id, it should be automatically generated 🙂